lib/radiator/directory/editor.ex

```diff
@@ -9,28 +9,26 @@ defmodule Radiator.Directory.Editor do
 
   alias Radiator.Repo
   alias Radiator.Directory
-  alias Directory.{Network, NetworkPermission}
-  alias Directory.{Podcast, PodcastPermission}
-  alias Directory.{Episode, EpisodePermission}
+  alias Directory.{Network, Podcast, Episode}
 
-  def get_permission(user, entity)
+  alias Directory.Editor.EditorHelpers
 
-  def get_permission(user = %Auth.User{}, episode = %Episode{}) do
-    case Repo.get_by(EpisodePermission, user_id: user.id, episode_id: episode.id) do
-      nil -> nil
-      perm -> perm.permission
-    end
+  def list_networks do
   end
 
-  def get_permission(user = %Auth.User{}, subject = %Podcast{}) do
-    case Repo.get_by(PodcastPermission, user_id: user.id, podcast_id: subject.id) do
-      nil -> nil
-      perm -> perm.permission
-    end
-  end
+  # Permission
+
+  def get_permission(user = %Auth.User{}, subject = %Network{}),
+    do: get_permission_p(user, subject)
+
+  def get_permission(user = %Auth.User{}, subject = %Podcast{}),
+    do: get_permission_p(user, subject)
+
+  def get_permission(user = %Auth.User{}, subject = %Episode{}),
+    do: get_permission_p(user, subject)
 
-  def get_permission(user = %Auth.User{}, subject = %Network{}) do
-    case Repo.get_by(NetworkPermission, user_id: user.id, network_id: subject.id) do
+  defp get_permission_p(user, subject) do
```

Comment:
The `_p` suffix is for "private"? The convention is to prefix the name with "do_" if you need to avoid name clash like here: `defp do_get_permission()`.

https://github.com/christopheradams/elixir_style_guide#private-functions-with-same-name-as-public

Reply:
Okay, I can go with that convention.